### PR TITLE
[Merged by Bors] - doc(Geometry): document a few items

### PR DESCRIPTION
**Pull request by grunweg**
Except for algebraic geometry and other in-flight PRs, this resolves all nolints in this directory.

-----------

The other PRs I'm talking about are
#12481 (items in EuclideanGeometry), #11905 (remaining notation) and #9827 (documents StructureGroupoid).

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Calculus/BumpFunction/Basic.lean [hunk 1]
```diff
@@ -66,6 +66,7 @@ The structure `ContDiffBump` contains the data required to construct the functio
 real numbers `rIn`, `rOut`, and proofs of `0 < rIn < rOut`. The function itself is available through
 `CoeFun` when the space is nice enough, i.e., satisfies the `HasContDiffBump` typeclass. -/
 structure ContDiffBump (c : E) where
+  /-- real numbers `0 < rIn < rOut` -/
```
Comment (urkud) [inline Mathlib/Analysis/Calculus/BumpFunction/Basic.lean:69]: Does it assign the same docstring to both `rIn` and `rOut`?

Comment (grunweg) [reply Mathlib/Analysis/Calculus/BumpFunction/Basic.lean:69]: Yes, it does. Given these fields are related, this actually felt nicer.
If you prefer the following phrasing, feel free to apply it directly (and manually delete the next line).
```suggestion
/-- real number -/
rIn : ℝ
/-- real number, larger than rIn -/
rOut : ℝ

Comment (urkud) [reply Mathlib/Analysis/Calculus/BumpFunction/Basic.lean:69]: No, mentioning both LGTM.

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**